x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/azure_credentials_form/azure_credentials_form.tsx

```diff
@@ -248,6 +248,18 @@ const TemporaryManualSetup = ({ integrationLink }: { integrationLink: string })
 const AZURE_MINIMUM_PACKAGE_VERSION = '1.6.0';
 const AZURE_MANUAL_FIELDS_PACKAGE_VERSION = '1.7.0';
 
+export const getDefaultAzureManualCredentialType = (packageInfo: PackageInfo) => {
+  const packageSemanticVersion = semverValid(packageInfo.version);
+  const cleanPackageVersion = semverCoerce(packageSemanticVersion) || '';
+
+  const isPackageVersionValidForManualFields = !semverLt(
+    cleanPackageVersion,
+    AZURE_MANUAL_FIELDS_PACKAGE_VERSION
+  );
+
+  return isPackageVersionValidForManualFields ? 'managed_identity' : 'manual';
```

Comment:
nit: I find it a bit confusing that when  "package is valid for manual" we return `managed identity` otherwise `manual`. without the context, I would think it should be the other way around. Maybe change to `isPackageVersionValidForManagedIdentity`?

Reply:
umm this is a very good point, thing is that "manual fields" are meant to reference the entire options (fields) under the Manual credential type, managed identity is just one of them and it happens to be the default value out of those options. 

do you think something like `isPackageVersionValidForManualOption` is better? i dont mind sticking with your suggestions, just wanted to give more context and see what do you think

Reply:
yeah, I see where the confusion is coming from, that we have Manual as a term that includes multiple setup methods, and inside it we have another `Manual` as its own setup method, as in "really manual, you need to do everything yourself" kind of thing. changing to `isPackageVersionValidForManualOption` tbh wouldn't change much for me, in the end starting from 1.7.0 we started to support `Manual -> Managed Identity` so it became the default, and before it was `Manual -> Manual` the default one. So `isPackageVersionValidForManagedIdentity` still makes more sense to me (even though it's not only Managed Identity but some other options we started to support) but I'm also ok with a comment in this function to explain this Manual extravaganza, without changing the name :)